src/App.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import { getThaAwwz, currentAww, selectAww } from './actions'
+import Awwz from './Awwz.js'
+import BigAww from './BigAww.js'
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    this.getMoar = this.getMoar.bind(this)
+  }
+
+  componentDidMount() {
+    const { dispatch, awwz } = this.props
+    dispatch(getThaAwwz(awwz))
+  }
+
+  componentWillReceiveProps() {
+  }
+
+  getMoar () {
+  }
+
+  render () {
+    const { awwz, currentAww, isFetching, dispatch } = this.props
+
+    return (
+      <div>
+        <BigAww aww={currentAww} />
+        {isFetching && awwz.length === 0 &&
```

Comment:
I'm not a fan of 
```
{ cond1 && cond2 && <div>hey</div> }
```

Some alternatives I think are preferable are:
```
render () {
  let body;
  if (!isFetching) {
    if (awwz.length > 0) {
      body = <Awwz awwz={...} />;
    } else {
      body = <h2>NO CAN HAZ</h2>;
    }
  } else {
    body = <h2>IZ LOADING</h2>
  }

  return (
    <div>
      { body }
    </div>
  );
}
```

```
render () {
  return (
    <div>
      { this.renderBody(isFetching, awwz) }
    </div>
  );
}

renderBody (isFetching, awwz) {
  // ...
}
```

The second way is more in line with what we're doing elsewhere, see `renderSegmentInfo` for an example